web3.js/src/transaction.ts

```diff
@@ -17,6 +17,22 @@ import type {CompiledInstruction} from './message';
  */
 export type TransactionSignature = string;
 
+export type TransactionBlockhash = {
+  blockhash?: Blockhash;
+  lastValidBlockHeight?: number;
+};
+
+export type TransactionSignatureBlockhash = {
+  signature: TransactionSignature;
+  blockhash?: Blockhash;
+  lastValidBlockHeight?: number;
+};
+
+export enum TransactionStatus {
+  CONFIRMED = 'CONFIRMED',
+  EXPIRED = 'EXPIRED',
+}
```

Comment:
If you use `const` enums with no values, TypeScript can optimize them right down to single digit integers.

<img width="904" alt="image" src="https://user-images.githubusercontent.com/13243/165632982-b420b329-0eb7-4387-bd7d-aa7e77fb70ee.png">